app/Infrastructure/Exceptions/Handler.php

```diff
@@ -55,6 +56,12 @@ public function render($request, Throwable $exception): Response
             ], Response::HTTP_NOT_FOUND);
         }
 
```

Comment:
We are duplicating codee!!!!!!
If the responses of the two exceptions are equals, maybe we can combinate?